distributed/http/worker/prometheus/core.py

```diff
@@ -52,6 +52,12 @@ def collect(self):
             value=self.server.latency,
         )
 
+        yield GaugeMetricFamily(
+            self.build_name("comm_reserved_bytes"),
```

Comment:
`comm_reserved_bytes` feels like a more descriptive name than `comm_nbytes` in general, I'm inclined to switch that out in general.

Reply:
please rename to `comm_incoming_nbytes` everywhere

Reply:
Please also update documentation at worker_state_machine.py:1143

Reply:
To clarify: Does this include renaming `WorkerState.comm_nbytes` (maybe with a deprecation warning)?